rest-api/api/participant_counts_over_time_api.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+# import json
+# import logging
+
+from flask.ext.restful import Resource
+from flask import request
+from werkzeug.exceptions import BadRequest
+
+from api_util import HEALTHPRO
+from api_util import get_awardee_id_from_name
+from app_util import auth_required
+from dao.hpo_dao import HPODao
+from participant_enums import EnrollmentStatus
+
+DATE_FORMAT = '%Y-%m-%d'
+DAYS_LIMIT = 100  # provisional, per design doc
```

Comment:
If you prefer we can make this a config setting (that can be changed without pushing new code); but this is fine, too.

Reply:
I'm fine keeping this as is for now, since it's consistent in form with [metrics_api.py](https://github.com/all-of-us/raw-data-repository/blob/306ab26977b2fe2594c0344fdde08bd091557ae2/rest-api/api/metrics_api.py#L12).